client/core/core.go

```diff
@@ -1765,6 +1779,48 @@ func (c *Core) CreateWallet(appPW, walletPW []byte, form *WalletForm) error {
 	return nil
 }
 
+// createSeededWallet initializes a seeded wallet with an asset-specific seed
+// derived deterministically from the app seed and a random password. The
```

Comment:
Outdated comment? Don't see a random password being used in deriving the asset-specific seed. Also makes me wonder how deterministic it is if the derivation uses a randomly generated input.

Reply:
This refers to the `pw := encode.RandomBytes(32)` fed into `CreateWallet` with the deterministic seed as well.  The `pw` is arbitrary, for whatever purposes the wallet backend needs it for, be it encryption or whatetever, while the seed is the deterministic part for key/address generation.

Reply:
The phrasing is just confusing.

Reply:
TBH, the bit about the password is not the concern of the caller of this function, and thus not totally necessary to be in the docs for the function.  The random password can be explained in the code if needed.